Popov_XO_test/Popov_XO_test/main.cpp

```diff
@@ -0,0 +1,70 @@
+#include "vector"
+#include "TreeNode.h"
+#include "string"
+#include "iostream"
+
+using namespace std;
+
+void PrintField(PlayField field) {
```

Comment:
```suggestion
void PrintField(const PlayField &field) {
```
Нафига копию делать?